src/modules/mavlink/mavlink_stream.cpp

```diff
@@ -312,6 +322,8 @@ MavlinkStreamPoll::unregister_orbs(uint16_t stream_id)
 int
 MavlinkStreamPoll::set_interval(uint16_t stream_id, int interval_ms)
 {
+	pthread_mutex_lock(&_mtx);
+
```

Comment:
The comment below this line is nonsense / leftovers from some old stuff. Just writing a note of it here, not related to this PR

Reply:
fixed